modules/libcom/src/osi/os/WIN32/osdProcess.c

```diff
@@ -50,10 +50,14 @@ LIBCOM_API osiGetUserNameReturn epicsStdCall osiGetUserName (char *pBuf, unsigne
 LIBCOM_API osiSpawnDetachedProcessReturn epicsStdCall osiSpawnDetachedProcess 
     ( const char *pProcessName, const char *pBaseExecutableName )
 {
+    BOOL silent = pProcessName[0]=='!';
```

Comment:
Since `osiSpawnDetachedProcess()` is a public API, should this instead be:
```C
BOOL silent = pProcessName && pProcessName[0]=='!';
```
Windows permits a NULL process name [`STARTUPINFO.lpTitle`](https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/ns-processthreadsapi-startupinfoa)